src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length; j++) {
+        if (compareFunction(this[j], this[j + 1]) > 0) {
+          const sort = this[j];
+
+          this[j] = this[j + 1];
+          this[j + 1] = sort;
```

Comment:
You can use array destructuring to swap elements, which is a more modern and concise way to write this. For example: `[this[j], this[j + 1]] = [this[j + 1], this[j]]`.